Cargo.lock

```diff
@@ -560,15 +560,15 @@ dependencies = [
 
 [[package]]
 name = "carrot"
-version = "0.4.1"
+version = "0.5.0"
 dependencies = [
  "actix-codec 0.2.0",
  "actix-multipart",
  "actix-multipart-rfc7578",
  "actix-rt",
  "actix-service",
  "actix-web",
- "base64 0.13.0",
+ "base64 0.9.3",
```

Comment:
Why did this go down a version?

Reply:
That's a very good question, actually.  It shouldn't cause an issue, looking at the changes since then.  But it looks like I'm not specifying a specific version for that and I probably should be, so I'll make that change in a new branch.